### ISSUE TITLE
Background

### ISSUE THREAD

**Issue by Jasmineb2011** [lotus-1/To-do-Project-SES#14]
When the client adds something to his list, the background zooms in, and it shouldn't be.
The default view:
![Screenshot from 2019-04-11 21-19-51](https://user-images.githubusercontent.com/45990980/55981579-b3f14600-5c9f-11e9-8666-6d9aa883c1b6.png)
By adding Items:
![Screenshot from 2019-04-11 21-21-49](https://user-images.githubusercontent.com/45990980/55981642-d97e4f80-5c9f-11e9-8cf9-4067dd15ab75.png)

**Comment by enasskmal** [comment]
hii jasmine , yes it's zooming. So you can see the screen better.

**Comment by MynahMarie** [comment]
I agree with @Jasmineb2011 , it makes the design of the page confusing and the font of the todos get lost because of the background image. There should be a way to improve this.
